src/main/java/domain/NumberGenerator.java

```diff
@@ -0,0 +1,37 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class NumberGenerator {
+
+    public static final int FIRST_NUMBER = 1;
+    public static final int LAST_NUMBER = 45;
+    public static final int INT_ZERO = 0;
+    public static final int INT_SIX = 6;
+
+    private List<Integer> lottoNumberCandidate;
+
+    public NumberGenerator() {
+        lottoNumberCandidate = new ArrayList<>();
+        createLottoNumberCandidate();
+    }
+
+    private void createLottoNumberCandidate() {
+        lottoNumberCandidate = IntStream.rangeClosed(FIRST_NUMBER, LAST_NUMBER)
+                .boxed().collect(Collectors.toList());
+    }
```

Comment:
```suggestion
    private List<Integer> createLottoNumberCandidate() {
        return IntStream.rangeClosed(FIRST_NUMBER, LAST_NUMBER)
                .boxed().collect(Collectors.toList());
    }
```
리스트를 반환하는 구조로 변경하는 구조도 괜찮을 것 같네요:)

Reply:
네 제안하신 부분이 더 깔끔한것같네요. 수정했습니다 :)